packages/manager/modules/pci/src/projects/project/public-gateways/delete/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_public_gateway_delete": "Supprimer la Public Gateway",
+  "pci_projects_project_public_gateway_delete_confirmation": "Souhaitez-vous réellement supprimer la Public Gateway {{name}} ?",
+  "pci_projects_project_public_gateway_delete_confirm": "Confirmer",
+  "pci_projects_project_public_gateway_delete_cancel": "Annuler",
+  "pci_projects_project_public_gateway_delete_success": "Le Public Gateway {{name}} a bien été supprimée.",
+  "pci_projects_project_public_gateway_delete_error": "Une erreur est survenue lors de la suppression du Public Gateway {{name}}"
```

In previous phrases we consider `la Public Gateway` as female and here as male.
Better like this i guess or change in others phrase to consider `Public Gateway` as male :
```suggestion
  "pci_projects_project_public_gateway_delete_error": "Une erreur est survenue lors de la suppression de la Public Gateway {{name}}"
```

Ask PU to confirm

---

confirmed from PU and updated.